coding-concepts.js

```diff
@@ -7,44 +7,44 @@
 
 // --------------------1) What will this log?
 
-var cohort = "Charlie 2021"
+// var cohort = "Charlie 2021"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["C", "h", "a", "r", "l", "i", "e", "2", "0", "2", "1"]
+// b) Verify and explain: The .split method turns a string into an array of substrings.
 
 
 // --------------------2) What will this log?
-
-const greeter = (name) => {
-  `Hello, ${name}!`
-}
+//
+// const greeter = (name) => {
+//   `Hello, ${name}!`
+// }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log undefined
+// b) Verify and explain: It logged undefined because "Learn Student" was not defined
```

Comment:
"LEARN student" doesn't need to be defined to be passed as a string. What else is happening here?